hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/states/ContainerEntry.java

```diff
@@ -31,8 +31,8 @@
  */
 public class ContainerEntry {
   private final ContainerInfo info;
-  private final Map<DatanodeID, ContainerReplica> replicas = new HashMap<>();
-
+  private Map<ContainerReplica, ContainerReplica> replicaMap = Collections.emptyMap();
+  private Set<ContainerReplica> replicas = Collections.emptySet();
   ContainerEntry(ContainerInfo info) {
```

Comment:
Please make sure to check [CI run](https://github.com/cchung100m/ozone/actions/runs/19934514493) in fork.  Checkstyle is [failing](https://github.com/cchung100m/ozone/actions/runs/19934514493/job/57155484251#step:16:17) with:

```
hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/states/ContainerEntry.java
 36: 'CTOR_DEF' should be separated from previous line.
```